### PR TITLE
Update DOCKER.md

### PR DESCRIPTION
**Pull request by dbolser**
s/v0.12.0-beta/v0.13.1-beta/

#### Pull Request Checklist

- [ ] All changes are Go version 1.15 compliant
- [ ] Your PR passes all CI checks. If a check cannot be passed for a justifiable reason, that reason must be stated in the commit message and PR description.
- [ ] If this is your first time contributing, we recommend you read the [Code Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
   - [ ] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
   - [ ] Commits have a logical structure according to [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)
- [ ] For new code: Code is accompanied by tests which exercise both the positive and negative (error paths) conditions (if applicable)
- [ ] For bug fixes: If possible, code is accompanied by new tests which trigger the bug being fixed to prevent regressions
- [ ] Any new logging statements use an appropriate subsystem and logging level
- [ ] For code and documentation: lines are wrapped at 80 characters (the tab character should be counted as 8 characters, not 4, as some IDEs do per default)

### CODE REVIEW

**Comment by bhandras** [comment]
Thanks for the fixes @dbolser. Could you please squash these two commits into one, and also add another updating `docs/release-notes/release-notes-0.14.0.md`. Minor thing but the commit messages (after squasing what you have currently) should be something like:

`docs: update DOCKER.md`
`docs: update release-notes-0.14.0.md`

File: docs/DOCKER.md [hunk 1]
```diff
@@ -63,9 +63,7 @@ example):
 ```shell
 ⛰  docker pull lightninglabs/lnd:v0.13.1-beta
 ⛰  docker run --rm --entrypoint="" lightninglabs/lnd:v0.13.1-beta /verify-install.sh
-⛰  OK=$?
-⛰  if [ "$OK" -ne "0" ]; then echo "Verification failed!"; exit 1; done
-⛰  docker run lightninglabs/lnd [command-line options]
+⛰  docker run lightninglabs/lnd:v0.13.1-beta [command-line options]
```
Comment (bhandras) [inline docs/DOCKER.md:66]: please squash with prev commit

Comment (dbolser) [reply docs/DOCKER.md:66]: Sorry, must have missed this message way back! Let my try to do that :-)

**Comment by guggero** [comment]
Replaced by https://github.com/lightningnetwork/lnd/pull/6080 to speed up review process. Original commit author was preserved, thank you for your contribution.

**Pull request closed without merging**